src/SuperDump/Analyzers/WinDbgAnalyzer.cs

```diff
@@ -52,23 +52,17 @@ public class WinDbgAnalyzer {
 			LogOnErrorHR(debugControl.Execute(DEBUG_OUTCTL.ALL_CLIENTS, $"$$ {msg}", DEBUG_EXECUTE.DEFAULT), $"failed to " + $"$$ {msg}");
 		}
 
-		private static void LoadExtensions(IDebugControl6 debugControl) {
-			if (Environment.Is64BitProcess) {
-				LoadExtension(debugControl, @"C:\Program Files (x86)\Windows Kits\10\Debuggers\x64\winext\ext.dll"); // do we need this configurable? should we ship these?
-				LoadExtension(debugControl, @"C:\Program Files (x86)\Windows Kits\10\Debuggers\x64\WINXP\exts.dll");
-				LoadExtension(debugControl, @"C:\Program Files (x86)\Windows Kits\10\Debuggers\x64\WINXP\uext.dll");
-				LoadExtension(debugControl, @"C:\Program Files (x86)\Windows Kits\10\Debuggers\x64\WINXP\ntsdexts.dll");
-			} else {
-				LoadExtension(debugControl, @"C:\Program Files (x86)\Windows Kits\10\Debuggers\x86\winext\ext.dll"); // do we need this configurable? should we ship these?
-				LoadExtension(debugControl, @"C:\Program Files (x86)\Windows Kits\10\Debuggers\x86\WINXP\exts.dll");
-				LoadExtension(debugControl, @"C:\Program Files (x86)\Windows Kits\10\Debuggers\x86\WINXP\uext.dll");
-				LoadExtension(debugControl, @"C:\Program Files (x86)\Windows Kits\10\Debuggers\x86\WINXP\ntsdexts.dll");
-			}
-		}
+        private static void LoadExtensions(IDebugControl6 debugControl)
+        {
```

Comment:
just a minor thing. please bear with the code style which is used throughout the library, which is "Tabs not Spaces" and "{ on the same line".